addon/styles/addon.css

```diff
@@ -1,5 +1,5 @@
 .ember-light-table {
-  height: auto;
+  height: inherit;
```

Comment:
Setting this to inherit, allow the table to get its height from the parent container